torch/fx/passes/annotate_getitem_nodes.py

```diff
@@ -23,7 +23,7 @@ def annotate_getitem_nodes(graph: torch.fx.Graph) -> None:
             # container types
             if hasattr(sequence_node.type, "_name"):
                 parameterized_types = sequence_node.type.__args__
-                if sequence_node.type._name == "Tuple":
+                if sequence_node.type._name in ("tuple", "Tuple"):
```

Comment:
We need to leave a few tests / ads a few to test the legacy imports / import aliases. Otherwise, only the new import path tests will be exercised here.

Reply:
Will add old and new annotation tests when test_fx_experimental.py is changed